crates/db_views_actor/src/person_view.rs

```diff
@@ -51,9 +51,20 @@ fn queries<'a>(
   let all_joins = |query: person::BoxedQuery<'a, Pg>| {
     query
       .inner_join(person_aggregates::table)
-      .left_join(local_user::table)
+      // This will only join if the admin is true, so you can use is_not_null() below
+      .left_join(
+        local_user::table.on(
+          person::id
+            .eq(local_user::person_id)
+            .and(local_user::admin.eq(true)),
+        ),
+      )
```

Comment:
you're still using the left_join instead of exists method here in person_view, intentional?

Reply:
My bad, good catch. I should've grepped for left_join.